metricbeat/module/elasticsearch/metricset.go

```diff
@@ -0,0 +1,58 @@
+package elasticsearch
+
+import (
+	"github.com/elastic/beats/libbeat/common/cfgwarn"
+	"github.com/elastic/beats/metricbeat/helper"
+	"github.com/elastic/beats/metricbeat/mb"
+	"github.com/elastic/beats/metricbeat/mb/parse"
+)
+
+const (
+	defaultScheme = "http"
+	pathConfigKey = "management_path_prefix"
```

Comment:
The `pathConfigKey` was `path` before. I used `management_path_prefix` in #7074 because is the terminology in RabbitMQ.

Reply:
Ups, this is a leftover. Changed to `path`.